databuilder/models/user_elasticsearch_document.py

```diff
@@ -0,0 +1,62 @@
+import json
+from typing import List, Optional  # noqa: F401
+
+from databuilder.models.elasticsearch_document import ElasticsearchDocument
+
+
+class UserESDocument(ElasticsearchDocument):
+    """
+    Schema for the Search index document for user
+    """
+    def __init__(self,
+                 elasticsearch_index,  # type: str
+                 elasticsearch_type,   # type: str
+                 email,  # type: str
+                 first_name='',  # type: str
```

Comment:
Should we keep it `None` instead of empty string and not passing `None` value to ES JSON?

Reply:
I think it will be an issue the document is exposed though API, then different user will have different attribute which could be an issue.

Reply:
Mmm. Could you elaborate? Example would be great :)

Reply:
@jinhyukchang , remove default argument assignment.